### ISSUE TITLE
Duplicate creation of Gantt tasks in real-time when trying to edit the newly created task 

### ISSUE THREAD

**Issue by maartenraes** [bryntum/support#10477]
Probrably related to https://bryntum.com/products/gantt/docs/guide/Gantt/revisions/revisions#handling-record-duplicates where an add followed by an update is resolved to 2 adds. 

> When a client adds and immediately modifies a new record, resulting in two local revisions, the server must identify and handle duplicate records. It should update existing records with the same Phantom ID instead of creating new ones, necessitating tracking of Phantom IDs and real IDs for all records.

### Repro case: 
1. Open https://bryntum.com/products/gantt/examples/realtime-updates/
2. Login to both instances
3. Scroll down to 'Celebrate launch' task
4. Double click to open cell edit
5. Press enter -> this should create a new task
6. Without leaving the cell, update the new task to 'foo'
7. Trigger onBlur by clicking outside the cell
8. -> Unexpected result that 'foo' was created twice

### Expected behavior
Foo should be created, but only once
